src/main/kotlin/com/hibob/academy/resource/petsResource.kt

```diff
@@ -17,22 +18,26 @@ class PetsResource {
     @Consumes(MediaType.APPLICATION_JSON)
     fun createPet(@RequestBody pets: Pets): Response {
         Response.status(Response.Status.OK).build()
-        return Response.ok("POST OK").build()
+        return Response.ok(pets).build()
     }
 
     @GET
     @Path("/{petId}/type")
     fun getPetType(@PathParam("petId") id: String): Response {
         Response.status(Response.Status.OK).build()
-        return Response.ok("GET OK").build()
+        return Response.ok(listOf(Pets(name = "dog", type = "lab"
+            , company_id = UUID.randomUUID(), date_of_arrival = Date() ))).build()
```

Comment:
I personally prefer it this way:
```suggestion
        return Response.ok(listOf(Pets(name = "dog", type = "lab",
            company_id = UUID.randomUUID(), date_of_arrival = Date()))).build()
```

Reply:
also, if this API is for getting a type, I would expect this to return a type and not list of pets